scipio/src/executor.rs

```diff
@@ -150,7 +150,7 @@ impl Eq for TaskQueue {}
 impl TaskQueue {
     fn new<F>(
         index: usize,
-        name: &'static str,
+        name: &str,
```

Comment:
Mostly a style nit, but here and elsewhere I'd take a `String` as an argument, rather then implicitly allocating: https://github.com/rust-analyzer/rust-analyzer/blob/dedfaa384408b266c461fea45fee936989bb151b/docs/dev/style.md#push-allocations-to-the-call-site

Reply:
this makes sense, actually.

Will change, thanks!

Reply:
changed to use `Into<String>` which solves the allocation problem without forcing an API change.